common.mk

```diff
@@ -41,8 +41,8 @@ LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo.Version
 
 # Add supported OS-ARCHITECTURE combinations here
 ENVS ?= linux-amd64 windows-amd64 darwin-amd64
-STANDALONE_PLUGINS ?= login management-cluster package pinniped-auth secret
-CONTEXTAWARE_PLUGINS ?= cluster kubernetes-release feature
```

Comment:
Could you explain why this is not a context-aware plugin? [This comment](https://github.com/vmware-tanzu/tanzu-framework/pull/889#discussion_r769891212) says it should be one.

Reply:
yes, it could be a context-aware plugin.. but we need to do some thinking before making it a context-aware plugin, also just marking it as a context-aware plugin in the common.mk wouldn't make it a context-aware plugin(learned about it a couple of days ago), we need to add CLI plugin CR for feature plugin in https://github.com/vmware-tanzu/tanzu-framework/tree/main/packages/management/core-management-plugins and also do some plumbing on the bolt side..  in order to not ship broken feature plugin in the weekly release, I thought of changing this to standalone plugin for now and make it a context-aware plugin later.. let me know what you think?